Scoop/Supporting/PickerOptions.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  PickerOptions.swift
+//  Scoop
+//
+//  Created by Richie Sun on 10/11/23.
+//
+
+import Foundation
+
+class PickerOptions {
+    
+    static let pronouns = ["He/Him", "She/Her", "They/Them", "Other"]
+    static let years = ["Freshman", "Sophomore", "Junior", "Senior", "Grad Student"]
+    
+}
```

Comment:
Lowkey I would create a singleton. Then use `PickerOptions.scooped.pronouns` something like that. idk if the code below works but u get the idea
```suggestion
class PickerOptions {

   static let scooped = PickerOptions()
    
   let pronouns = ["He/Him", "She/Her", "They/Them", "Other"]
   let years = ["Freshman", "Sophomore", "Junior", "Senior", "Grad Student"]
    
}
```